backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -1432,14 +1433,3 @@ databaseChangeLog:
               SET device_specimen_type_id = ds.internal_id
               FROM ${database.defaultSchemaName}.device_specimen_type ds
               WHERE ds.device_type_id = t.device_type_id;
-  - changeSet:
```

Comment:
This is obviously not a long-term solution by itself, but it won't leave us any worse off than we already are.